perl/lib/Seq/Output.pm

```diff
@@ -210,23 +246,30 @@ sub makeOutputString {
           # t1;t2 \t t1_name1\\t1_name2;t2_onlyName
           $posData = join(
```

Comment:
in fact it looks like this whole expression could be pulled out into a helper?

I notice there's a slight difference between these two expressions: the first assigns to `$row->[$oIdx][$featIdx]` a function of `$row->[$oIdx][$featIdx][0]`, and the second assigns to `$posData` the same function of `$posData`.    I am probably missing context here but that almost-but-not-exactly-identical expressions might benefit from a comment explaining the difference.

Reply:
So 99% of sites won't have positional data, because only roughly 1% of all mutations are indels. It turns out that not entering the loop over position indices saves something like 20% runtime (tested on a small run, will benchmark more in depth on larger ones later).

Reply:
Also, we save a large number of allocations by directly mutating the array, which in perl you can do as seen because $posData in `for my $posData (@data)` is a reference to the array entry.